scripts/run_olserver.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This script git pulls master for openlibrary, infogami, and olsystem
+# and then starts the Docker service specified by $SERVICE
+
+cd /opt/olsystem
+sudo git pull origin master
+cd /opt/openlibrary
+sudo git pull origin master
+cd /opt/openlibrary/vendor/infogami
+sudo git pull origin master
+cd /opt/openlibrary
```

Comment:
Don't need this again (you do it later)

```suggestion
```

Reply:
I was thinking that folks would break this script in two halves in order to test non-master branches.  It is a common source of docker-compose errors to be in the wrong dir.